src/lc-tlscert/lc-tlscert.go

```diff
@@ -75,6 +76,18 @@ func anyKey() {
 
 func main() {
 	var err error
+	var common_name string
+	var days string
+	var cert_base string
+	var omitConfirm bool
+	flag.StringVar(&common_name, "name", "", "The common name")
+	flag.StringVar(&cert_base, "p", "selfsigned", "The base path of the certs")
+	flag.StringVar(&days, "days", "", "The common name")
+	flag.BoolVar(&omitConfirm, "y", false, "continue to generate certs w/o waiting for confirmation")
+	flag.Parse()
+	dnsIPs := flag.Args()
```

Comment:
I think the problem here is you could specify one or two of the options, but not the others, and then you get prompted.
Potentially not a major issue but it does mean that if we add an option down the line, scripts hang rather than fail.

I think best would be to have a "batch" command line parameter (maybe to replace the omitConfirm) which triggers batch mode. This mode **requires** all information on the command line. Taking the first non-option argument as common name could work best too to simplify the arguments. Maybe days could also be optional and have a default value of 365?

cert_base is better called target_dir too

I'd also rather it be refactored so that we only have one control flow logic for batch versus no batch. Likely means separate functions for populating the certificate information, but once that's done the code is the same.

Other than that I like the idea and I don't mind taking over from here.